itenium-socks/src/app/app.component.ts

```diff
@@ -9,4 +10,10 @@ import { RouterOutlet } from '@angular/router';
   ],
   templateUrl: './app.component.html',
 })
-export class AppComponent {}
+export class AppComponent implements OnInit {
+  isProduction!: boolean;
```

Comment:
fyi (don't update this)

You can do:
```
isProduction = environment.production;
```

Reply:
zo dan? 

@Component({
  selector: 'app-root',
  standalone: true,
  imports: [
    RouterOutlet,
  ],
  templateUrl: './app.component.html',
})
export class AppComponent{
  isProduction = environment.production;
}

Reply:
Weer 4 lijnen code minder ;)